RecoTracker/FinalTrackSelectors/plugins/getBestVertex.h

```diff
@@ -2,11 +2,9 @@
 #include "DataFormats/VertexReco/interface/Vertex.h"
 #include <limits>
 
-namespace {
```

Comment:
is this removal really harmless?
Would there be nothing else to include this file and have a possibly different meaning of Point?